.github/ISSUE_TEMPLATE/fix-content.yml

```diff
@@ -0,0 +1,34 @@
+name: Fix Content
+description: Report typos or minor documentation issues
+title: '[FIX]: '
+labels: ['update']
```

Comment:
I actually don't think we need both a default title tag and a label. One or the other should suffice.